projects/layout/components/cell/cell.options.ts

```diff
@@ -0,0 +1,16 @@
+import type {Provider} from '@angular/core';
+import {tuiCreateToken, tuiProvideOptions} from '@taiga-ui/cdk/utils/miscellaneous';
+import type {TuiSizeL, TuiSizeS} from '@taiga-ui/core/types';
+
+export interface TuiCellOptions {
+    readonly height: 'compat' | 'default' | 'spacious';
+    readonly size: TuiSizeL | TuiSizeS | '';
+}
+
+export const TUI_CELL_DEFAULT_OPTIONS: TuiCellOptions = {height: 'default', size: ''};
```

Comment:
Size should be `'l'`, input supports empty line just so you do not have to always specify the size.

Reply:
fixed